pkg/cmd/create/create.go

```diff
@@ -0,0 +1,68 @@
+package create
+
+import (
+	"github.com/spf13/cobra"
+
+	run "github.com/rancher/k3d/cli"
+	"github.com/rancher/k3d/pkg/constants"
+)
+
+type createOptions struct {
+	name           string
+	volume         []string
+	publish        []string
+	portAutoOffset int
+	version        string
+	apiPort        string
+	wait           int
+	image          string
+	serverArg      []string
+	agentArg       []string
+	env            []string
+	workers        int
+	autoRestart    bool
+}
+
+func NewCommand() *cobra.Command {
+	opts := &createOptions{}
+	// createCmd represents the create command
+	var createCmd = &cobra.Command{
+		Use:     "create",
+		Args:    cobra.NoArgs,
+		Short:   "Create a single- or multi-node k3s cluster in docker containers",
+		Aliases: []string{"c"},
+		RunE: func(_ *cobra.Command, _ []string) error {
+			return run.CreateCluster(
+				opts.name,
+				opts.image,
+				opts.env,
+				opts.apiPort,
+				opts.serverArg,
+				opts.agentArg,
+				opts.workers,
+				opts.publish,
+				opts.volume,
+				opts.autoRestart,
+				opts.portAutoOffset,
+				opts.wait,
+			)
+		},
+	}
+
+	createCmd.Flags().StringVarP(&opts.name, "name", "n", constants.DefaultK3sClusterName, "Set a name for the cluster")
+	createCmd.Flags().StringSliceVarP(&opts.volume, "volume", "v", []string{}, "Set a name for the cluster")
+	createCmd.Flags().StringSliceVar(&opts.publish, "publish", []string{}, "Publish k3s node ports to the host (Format: `[ip:][host-port:]container-port[/protocol]@node-specifier`, use multiple options to expose more ports)")
+	createCmd.Flags().IntVar(&opts.portAutoOffset, "port-auto-offset", 0, "Automatically add an offset (* worker number) to the chosen host port when using `--publish` to map the same container-port from multiple k3d workers to the host")
+	createCmd.Flags().StringVar(&opts.version, "version", "", "Choose the k3s image version")
```

Comment:
If we're going to deprecate lots of stuff with this PR, then we should as well deprecate the `--version` flag, I suppose :+1: